staging/src/k8s.io/csi-api/pkg/apis/csi/v1alpha1/types.go

```diff
@@ -99,6 +99,26 @@ type CSINodeInfo struct {
 
 // CSIDriverInfo contains information about one CSI driver installed on a node.
 type CSIDriverInfo struct {
+	// Specification of CSIDriverInfo
+	Spec CSIDriverInfoSpec `json:"spec"`
+
+	// Status of CSIDriverInfo
+	Status CSIDriverInfoStatus `json:"status"`
+}
+
+type CSIDriverInfoStatus struct {
+	// Installed is a boolean representing whether the driver has been installed
```

Comment:
s/Installed/`installed`/

Is `present` a better name? Perhaps `available`?

Reply:
using available